lib/CloudFormationDeployer.js

```diff
@@ -1,8 +1,18 @@
 module.exports = function CloudFormationDeployer(cloudFormationClient) {
 	this.cloudFormationClient = cloudFormationClient;
 
+	this.getTemplateBody = function(template) {
```

Comment:
For reference, it should take a string, and it seems this function is expecting an object.  Likely we can could just say
```
if (typeof(template) !== 'object') {
  return template;
}
return JSON.stringify(template, null, 2);
```